CONTRIBUTING.md

```diff
@@ -78,17 +79,15 @@ In addition to these, we also have several preferences:
       StringNumber = "fourty-two",
       Number = 42
   };
-  
+
   var b = new Dictionary<string, int>()
   {
       ["fourty-two"] = 42,
       ["sixty-nine"] = 69
   };
-  
-  var c = new List<int>() { 42, 69 };
-  
-  var d = new[] { 42, 69 };
-  ```
+
+  var c = [42, 69];
```

Comment:
is this even legal? at any rate, it breaks our style, and should specify the type

Reply:
...it would be legal if we didn't use `var`.